src/clojure/clojure/tools/nrepl/middleware/interruptible_eval.clj

```diff
@@ -211,7 +211,7 @@
       (case op
         "eval"
         (if-not (:code msg)
-          (t/send transport (response-for msg :status #{:error :no-code}))
+          (t/send transport (response-for msg :status #{:error :no-code :done}))
```

Comment:
Can we add a test for this change as well?

Reply:
Test added.